include/seqan3/io/views/detail/take_until_view.hpp

```diff
@@ -356,16 +352,12 @@ class view_take_until<urng_t, fun_t, or_throw, and_consume>::basic_sentinel
 private:
     //!\brief The sentinel type of the underlying range.
     using underlying_sentinel_t = seqan3::detail::maybe_const_sentinel_t<const_range, urng_t>;
-    //!\brief Auxiliary type.
-    using predicate_ref_t = std::conditional_t<const_range,
-                                               std::remove_reference_t<fun_t> const &,
-                                               std::remove_reference_t<fun_t> &>;
```

Comment:
why could you remove this?

Reply:
I'm not binding a reference any more, but a pointer, no I don't need to determine the correct reference type.

`copyable_wrapper` cannot be default constructed if the wrapped type is not default constructible (can't default construct a lambda).